src/coreclr/pal/src/file/filetime.cpp

```diff
@@ -263,6 +263,7 @@ BOOL PALAPI FileTimeToSystemTime( CONST FILETIME * lpFileTime,
 #else   /* HAVE_GMTIME_R */
         UnixSystemTime = gmtime( &UnixFileTime );
 #endif  /* HAVE_GMTIME_R */
+        if (!UnixSystemTime) return FALSE;
```

Comment:
```suggestion
        if (!UnixSystemTime)
        {
            ERROR( "gmtime failed.\n" );
            SetLastError(ERROR_INVALID_PARAMETER);
            return FALSE;
        }
```

Reply:
Jan, thanks for the correction, we'll take it into account for the future.

Reply:
Commit updated.

Reply:
Corrected